.github/workflows/trivy-sidecars.yaml

```diff
@@ -0,0 +1,49 @@
+name: Trivy Scanner - Container Images
```

Comment:
I think this file should be renamed to `trivy-containers.yaml` or similar now that we're not just scanning the sidecars